applications/FluidDynamicsApplication/custom_processes/distance_modification_process.cpp

```diff
@@ -483,6 +500,52 @@ void DistanceModificationProcess::SetDiscontinuousDistanceToSplitFlag()
     }
 }
 
+void DistanceModificationProcess::CheckAndStoreVariablesList(const std::vector<std::string>& rVariableStringArray) 
+{
+    const auto& r_node = *mrModelPart.NodesBegin(); 
+    if (rVariableStringArray.size()>0){
```

Comment:
What is `if` checking? If `size() == 0` the `for` loop in the next line should do 0 iterations.

Reply:
Yes you are right, the for loop would do 0 iterations, i can remove it.

There is no difference in performance right?

Reply:
I'd not expect to see a big difference, but in general it is a good practice to reduce branching (so the less if/else, the better).